config-model/src/main/resources/schema/admin.rnc

```diff
@@ -109,3 +111,17 @@ ClusterControllers = element cluster-controllers {
        service.attlist
     }+
 }
+
+LogForwarding = element logforwarding {
+    element forward {
+        attribute type {xsd:string } &
+        element source {
+            element log {  "access"| "vespa" }+
+        } &
+        element destination {
+            element endpoint { xsd:string } &
```

Comment:
Is this comma-separated? If so, I suggest instead to allow multiple 'endpoint' elements instead.

Reply:
Yes, this was discussed at architect meeting and ended up like this, I'm afraid.